src/routes/fest-route/fest-route.jsx

```diff
@@ -21,6 +32,20 @@ export const FestRoute = () => {
 
   const moviesQuery = useQuery(GET_MOVIES, { variables: { where: {} } });
 
+  const [deleteFest, { data, loading, error }] = useMutation(DELETE_FEST, {
+    refetchQueries: [GET_USER_FESTS],
+  });
+
+  const removeFest = () => {
+    if (currentUser.id === festQuery.data.fest.creator.id) {
+      deleteFest({ variables: { where: { id: festId } } });
+      if (loading) return "Submitting...";
+      if (error) return `Submission error! ${error.message}`;
+      closeModal("confirmation");
```

Comment:
`closeModal()` does not take any arguments

Reply:
Fixed, oversight on my end